### ISSUE TITLE
ERRO: Por favor, corrija o CEP digitado, ele não está correto.

### ISSUE THREAD

**Issue by rafaballack** [deivisonarthur/OSC-Magento-Brasil#145]
Realizei a instalação do PATCH de segurança:  (SUPEE-6285) não tenho certeza se esse erro começou apos a instalação do patch, mas um problemas quando digito o cep apresenta a mensagem: Por favor, corrija o CEP digitado, ele não está correto.

Mesmo colocando com o ( - ) o problema ocorre.

Estou utilizando o  1.7.0.2

**Comment by rafaelstz** [comment]
Se colocar um CEP diferente o erro persiste? Há alguma mascará no input?
Se remover o PATCH (passando o parâmetro -R) ele continuar com erro?

**Comment by rafaballack** [comment]
Sim, ainda continua do mesmo jeito. Qualquer cep o problema continua. Ja removi o modulo e instalei novamente ainda continua.
